LICENSE

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2017-2019 block.one and its contributors.  All rights reserved.
+Copyright (c) 2017-2019 CyberWay and its contributors.  All rights reserved.
```

Comment:
`2017-2020`

Reply:
@afalaleev It is 2019- only for me :(
For you it is 2018-. No?